apps/yuudachi/src/functions/anti-raid/blastOff.ts

```diff
@@ -21,6 +22,7 @@ export async function blastOff(
 
 	const confirmedHits: GuildMember[] = [];
 	for (const member of confirmations) {
+		await acquireMemberLock(member, locale);
```

Comment:
We should probably have this lock in the list of promises so that the rejections are handled gracefully without killing the entire nuke. Also, many of the mod actions may run significantly after this lock has been acquired, making it less useful.

Reply:
> We should probably have this lock in the list of promises so that the rejections are handled gracefully without killing the entire nuke.
Good idea

> Also, many of the mod actions may run significantly after this lock has been acquired, making it less useful.
That makes total sense, tbh I have my doubts about having this on the nuke

Reply:
Decided to remove the locks from anti-raid all together, they would be an relability nightbase based on the rate-limits from the operation itself